Gemfile

```diff
@@ -73,4 +73,6 @@ group :production do
   gem "pg"
 end
 
-gem 'dotenv'
\ No newline at end of file
+gem 'dotenv'
+
+gem 'rubocop', '~> 1.60', require: false
```

Comment:
You can also add rubcopo-rails and then require it in your `rubocop.yml`
then you can run just `rubcopo -a` or even `rubocop -A`

Reply:
I have added rubocop-rails and required it in my rubocop.yml also i run the rubocop -A to correct the present offenses.
Finally  i added  rubocop gems  to a development group